odilia/src/tower/handlers.rs

```diff
@@ -157,17 +145,12 @@ impl Handlers {
 		<T as TryFromState<Arc<ScreenReaderState>, E>>::Error: Send + 'static,
 		<T as TryFromState<Arc<ScreenReaderState>, E>>::Future: Send,
 	{
-		let ie_layer: AsyncTryIntoLayer<T, (Arc<ScreenReaderState>, E)> =
-			AsyncTryIntoLayer::new();
-		let state_layer: StateLayer<ScreenReaderState> =
-			StateLayer::new(Arc::clone(&self.state));
-		let ti_layer: TryIntoLayer<E, Request> = TryIntoLayer::new();
-		let ws = handler
+		let serv = handler
 			.into_service::<R>()
-			.map_result(|r| r.expect("This should never fail").try_into_commands());
-		let ie_serv = ie_layer.layer(ws);
-		let ch_serv = state_layer.layer(ie_serv);
-		let serv = ti_layer.layer(ch_serv);
+			.unwrap_map(|res| res.try_into_commands())
+			.request_async_try_from()
+			.with_state(Arc::clone(&self.state))
+			.request_try_from();
 		let dn = (
```

Comment:
does this particular variable name mean something I should be aware of?

Reply:
No. I've removed it entirely in the latest commits via `Chooser` and `ChooserStatic` traits.